docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
       - ./:/www
       - /var/run/docker.sock:/var/run/docker.sock
       # For dev purposes this simulates npm-link, since symbolic links on the mac won't work in the docker container
-      # - ../aries-controller/dist:/www/node_modules/aries-controller
+      - ../aries-controller/dist:/www/node_modules/aries-controller
```

Comment:
do we want to leave this line in there?  I feel the default behavior should rely on packages rather than npm link